packages/papaya/backend/sirius-components-papaya/src/main/java/org/eclipse/sirius/components/papaya/impl/PapayaFactoryImpl.java

```diff
@@ -541,6 +546,27 @@ public String convertInstantToString(EDataType eDataType, Object instanceValue)
         return super.convertToString(eDataType, instanceValue);
     }
 
+    /**
+     * <!-- begin-user-doc --> <!-- end-user-doc -->
+     *
+     * @generated NOT
+     */
+    public LocalDate createLocalDateFromString(EDataType eDataType, String initialValue) {
+        return LocalDate.parse(initialValue);
```

Comment:
Catch `DateTimeParseException` (and return null)? This will be invoked (among other cases) during the loading of an EMF resource, do we want an invalid syntax here to break/stop the loading of the whole resource.